core/unit_test/TestViewAPI.hpp

```diff
@@ -827,6 +827,17 @@ struct TestViewMirror
     ASSERT_EQ( a_org(5), a_h3(5) );
   }
 
+  template< class MemoryTraits >
+  void static test_mirror_no_initialize() {
+    Kokkos::View< double*, Layout, Kokkos::HostSpace > a_org( "A", 10 );
+    Kokkos::View< double*, Layout, Kokkos::HostSpace, MemoryTraits > a_h;
+    auto a_d = Kokkos::create_mirror_view( DeviceType(), a_h, Kokkos::WithoutInitializing );
+    
+    int equal_ptr_h_d = (a_h.data() == a_d.data()) ? 1 : 0;
+    constexpr int is_same_memspace = std::is_same< Kokkos::HostSpace, typename DeviceType::memory_space >::value ? 1 : 0;
+    
+    ASSERT_EQ( equal_ptr_h_d, is_same_memspace);
```

Comment:
Doesn't `ASSERT_EQ` work with `bool`?

Reply:
Yea, I did bool at first but did this for consistency with surrounding code with uses ints. Kindof pointless, but if we switch this to bool we might as well clean that up in the whole file(s).